src/rosetta-helpers.ts

```diff
@@ -205,14 +213,41 @@ function processEvents(events: DbEvent[], baseTx: BaseTx, operations: RosettaOpe
       case DbEventTypeId.NonFungibleTokenAsset:
         break;
       case DbEventTypeId.FungibleTokenAsset:
+        const tokenName = event.asset_identifier.split('::')[0];
+        const ftMetadata = await db.getFtMetadata(tokenName);
```

Comment:
We might run into issues here. FT metadata resolving is disabled by default (opt-in by env var), and when enabled, it's a "best effort" system that isn't guaranteed to resolve. If you check a look at the token metadata resolver code, it's essentially a try/catch with some retries. 

The metadata values being used here are _more_ likely to be available since they come from hitting the stacks-node RPC endpoints (as opposed to fetching JSON files at arbitrary endpoints), but it still has various failure modes.